src/jdk.jdi/share/classes/com/sun/tools/jdi/VirtualMachineManagerImpl.java

```diff
@@ -99,12 +99,7 @@ protected VirtualMachineManagerImpl() {
 
             try {
                 connector = connectors.next();
-            } catch (ThreadDeath x) {
-                throw x;
-            } catch (Exception x) {
-                System.err.println(x);
-                continue;
-            } catch (Error x) {
+            } catch (Exception | Error x) {
```

Comment:
Maybe this should just catch `Throwable`, although it is unclear to me why we would want to catch any exception here.

Reply:
@plummercj  these changes are not actually part of this PR but came from the upstream PR this was dependent on.